core/src/main/java/org/apache/accumulo/core/metadata/schema/Ample.java

```diff
@@ -392,6 +393,8 @@ interface TabletUpdates<T> {
 
     T putCloned();
 
+    T putTabletMergeability(TabletMergeability tabletMergeability);
```

Comment:
Steady time could be added in later PRs.  Just wondering how it will be persisted.

Reply:
My original plan was to persist everything as one value by adding the current manager time to the value. So basically to compute the mergability duration you would read the current SteadyTime then add the offset to it and store it and then later you could compare to the current time by taking a diff.

However, I was thinking more about it and I think that storing it as two separate values makes sense because it allows you to do better debugging (logigng) plus you can see extra information such as when the value was created. It also allows doing other things like update/replacing the original SteadyTime, etc and other metric calculations if we keep it separate so I'll change it.

Reply:
> My original plan was to persist everything as one value by adding the current manager time to the value. 

Ok I had thought this would store the exact duration specified by the user.  Summing the steady time and duration from the user would work.  There may be a slight advantage for keeping them separate for debugging purposes as mentioned.